recipes/smartg/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "smartg" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/smartg-{{ version }}.tar.gz
+  sha256: 6f6a81314da6a3fab92395d4c0bb9bb7b3c10420280f9e11439ecaf54dea2c08
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [osx] 
```

Comment:
Looks like this one can be noarch. What is failing on osx?
If you must skip it please see https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies on how to keep it noarch while skip a platform.